### PR TITLE
Akker

### PR DESCRIPTION
**Pull request by chrisakker**
### Proposed changes

Cleaning up Merge Text

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginxinc/nginx-k8s-loadbalancer/blob/main/CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation ([`README.md`](https://github.com/nginxinc/nginx-k8s-loadbalancer/blob/main/README.md) and [`CHANGELOG.md`](https://github.com/nginxinc/nginx-k8s-loadbalancer/blob/main/CHANGELOG.md))

### CODE REVIEW

File: docs/http/clusters.conf [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# NginxK8sLB HTTP configuration, for L7 load balancing
+# NGINX Loadbalancer for K8s HTTP configuration, for L7 load balancing
```
Comment (ciroque) [inline docs/http/clusters.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

File: docs/http/default-http.conf [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Nginx K8s Loadbalancer Solution
+# NGINX Loadbalancer for K8s Solution
```
Comment (ciroque) [inline docs/http/default-http.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

File: docs/http/nginx.conf [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Nginx K8s Loadbalancer Solution
+# NGINX Loadbalancer for K8s Solution
```
Comment (ciroque) [inline docs/http/nginx.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

File: docs/http/prometheus.conf [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# NginxK8sLB Prometheus configuration, for HTTP scraper page
+# NGINX Loadbalancer for K8s - Prometheus configuration, for HTTP scraper page
```
Comment (ciroque) [inline docs/http/prometheus.conf:1]: I think we should use the full name "Kubernetes" here

File: docs/tcp/default-tcp.conf [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# Nginx K8s Loadbalancer Solution
+# NGINX Loadbalancer for K8s Solution
```
Comment (ciroque) [inline docs/tcp/default-tcp.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

File: docs/tcp/nginx.conf [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# Nginx K8s Loadbalancer Solution
+# NGINX Loadbalancer for K8s Solution
```
Comment (ciroque) [inline docs/tcp/nginx.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

File: docs/tcp/nginxk8slb.conf [hunk 7]
```diff
@@ -1,22 +1,22 @@
-# NginxK8sLB Stream configuration, for L4 load balancing
+# NGINX Loadbalancer for K8s Solution - Stream configuration, for L4 load balancing
```
Comment (ciroque) [inline docs/tcp/nginxk8slb.conf:1]: I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes

**Review by ciroque — CHANGES_REQUESTED** [review]
To be consistent I think we should use the full name and avoid abbreviations. So in the project name it should be "[Kk]ubernetes" not k8s.

**Review by ciroque — APPROVED** [review]

**Comment by ciroque** [comment]
Smashed this in manually

**Review by sdutta9 — COMMENTED** [review]
Forgot to submit these two reviews on time. My bad.

**Pull request closed without merging**